orchagent/flexcounterorch.h

```diff
@@ -15,10 +15,14 @@ class FlexCounterOrch: public Orch
     void doTask(Consumer &consumer);
     FlexCounterOrch(swss::DBConnector *db, std::vector<std::string> &tableNames);
     virtual ~FlexCounterOrch(void);
+    bool getPortCountersState();
+    bool getPortBufferDropCountersState();
  
 private:
     std::shared_ptr<swss::DBConnector> m_flexCounterDb = nullptr;
     std::shared_ptr<swss::ProducerTable> m_flexCounterGroupTable = nullptr;
+    bool m_port_counter_enabled;
+    bool m_port_buffer_drop_counter_enabled;
```

Comment:
```suggestion
    bool m_port_counter_enabled{false};
    bool m_port_buffer_drop_counter_enabled{false};
```
And you can remove their initialization in ctor